coop/rchain/key-value-store/Key.scala

```diff
@@ -0,0 +1,81 @@
+/*               __          __                                         *\
+**    __________/ /_  ____ _/_/___                                      **
+**   / ___/ ___/ __ \/ __ `/ / __ \     RChain API                      **
+**  / /  / /__/ / / / /_/ / / / / /     (c) http://rchain.coop          **
+** /_/   \___/_/ /_/\____/_/_/ /_/                                      **
+\*                                                                      */
+
+package KeyValueStore
+
+
+// Keys are like Prolog terms, where the predicate
+// name is all letters and each parameter is
+// all digits or all letters.
+
+class Key(key: String)
+{
```

Comment:
I know you asked about this before and I said to follow Greg's style, but since he's not directly contributing to the code, I think we should follow the official style guide here:
http://docs.scala-lang.org/style/
which has an executable spec here:
http://scalameta.org/scalafmt/